examples/manageddisk_lustre/config.json

```diff
@@ -0,0 +1,257 @@
+{
+  "location": "westeurope",
+  "resource_group": "variables.resource_group",
+  "install_from": "headnode",
+  "admin_user": "hpcadmin",
+  "vnet": {
+    "name": "hpcvnet",
+    "address_prefix": "10.2.0.0/20",
+    "subnets": {
+      "compute": "10.2.0.0/22",
+      "storage": "10.2.4.0/24"
+    }
+  },
+  "variables": {
+    "resource_group": "azhpc-cluster",
+    "lustre_disk_type": "data_disk",
+    "image": "OpenLogic:CentOS:7.6:latest",
+    "hpcimage": "OpenLogic:CentOS-HPC:7.6:latest",
+    "storage_account": "xxxxxxxx",
+    "storage_key": "/xxxxxxxxxxxxx",
+    "storage_container": "xxxxxxxxx",
+    "log_analytics_lfs_name": "xxxxxxx",
+    "log_analytics_workspace": "xxxxxxxx",
+    "log_analytics_key": "xxxxxxx",
+    "location": "westeurope"
+  },
+  "resources": {
+    "headnode": {
+      "type": "vm",
+      "vm_type": "Standard_HB60rs",
+      "accelerated_networking": false,
+      "public_ip": true,
+      "image": "variables.hpcimage",
+      "subnet": "compute",
+      "tags": [
+        "disable-selinux",
+        "lfsclient",
+        "lfsazimport",
+        "localuser",
+        "pbsserver",
+        "loginnode",
+        "nfsserver"
+      ]
+    },
+    "compute": {
+      "type": "vmss",
+      "vm_type": "Standard_HB60rs",
```

Comment:
Do you really need HB as an example for mounting Lustre ?

Reply:
This is what was done for Lustre previously with Azure. Either we change all of them or none of them I would think.

Reply:
Please start by changing this one, we will change the others too.